config/containerbuddy/reload-nginx.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SERVICE_NAME=${SERVICE_NAME:-nginx}
+
+if [ -z "$VIRTUALHOST" ]; then
+    # fetch latest virtualhost template from Consul k/v
+    curl -s --fail consul:8500/v1/kv/${SERVICE_NAME}/template?raw > /tmp/virtualhost.ctmpl
```

Comment:
Per our discussion, for now we'll leave this but circle back once Containerbuddy supports environment variable interpolation.